Tasks/Suprun/Task6/task6_5.py

```diff
@@ -0,0 +1,5 @@
+def fibonacci(n):
+    if n in (1, 2):
+        return 1
```

Comment:
are you one of those "witnesses of 1,1" who think that Fibonacci sequence starts with 1,1 instead of 0,1,1?)))